k3s-io/k3s
# Changes local storage pods to have 700 permissions

Signed-off-by: dereknola <derek.nola@suse.com>




#### Proposed Changes ####
Individual storage pods under `/var/lib/rancher/k3s/storage` will now have 0700 permissions to prevent other users from accessing them. 

#### Types of Changes ####
Changed mkdir call in `local-storage.yaml`

#### Verification ####
- Start k3s server
- Create local storage by following rancher guide https://rancher.com/docs/k3s/latest/en/storage/
- Verify that created storage under  `/var/lib/rancher/k3s/storage/` has 0700 permissions


#### Linked Issues ####
https://github.com/k3s-io/k3s/issues/2348
